apps/web/src/pages/Pools/EditPoolPage/components/WorkTasksSection.tsx

```diff
@@ -38,7 +38,8 @@ interface WorkTasksSectionProps {
   onSave: (submitData: WorkTasksSubmitData) => void;
 }
 
-const TEXT_AREA_MAX_WORDS = 400;
+const TEXT_AREA_MAX_WORDS_EN = 400;
+const TEXT_AREA_MAX_WORDS_FR = TEXT_AREA_MAX_WORDS_EN + 100;
```

Comment:
I'm not sure if this was a typo but the figma included the following:
- **English**: 300
- **French**: 440